spec/features/dams_object_spec.rb

```diff
@@ -1626,3 +1626,32 @@
     expect(page).to have_css('img.dams-search-thumbnail[src="https://library.ucsd.edu/assets/dams/site/thumb-restricted.png"]')
   end
 end
+
+describe "User wants to view an Image object" do
```

Comment:
Could we maybe add a complex object test as well? Or is that already covered elsewhere? Same for audio and video examples.

Reply:
@mcritchlow - I can add complex object for image test but not audio and video.  Since we have the javascript problem for new jwplayer. That's why I marked some tests as pending before.  For example - https://github.com/ucsdlib/damspas/blob/master/spec/features/dams_object_spec.rb#L696

Reply:
Ah that's right, I remember talking about that. Ok nevermind then :)

Reply:
@mcritchlow - A test for complex object, width and height has been added. Could you review again?  Thanks